2/src/components/__tests__/List.spec.js

```diff
@@ -0,0 +1,28 @@
+import { shallowMount } from '@vue/test-utils'
+import List from '../List.vue'
+import ListItem from '../ListItem.vue'
+
+const factory = () => {
+	return shallowMount(List)
+}
+
+describe('List', () => {
+	const wrapper = factory();
```

Comment:
But you should re-initialize the wrapper with your setup method. Either you repeat the one-liner `wrapper = factory()` in each test case (which is not bad) or you use a `beforeEach` to setup the wrapper.